Sources/View/ReusableView.swift

```diff
@@ -20,15 +20,19 @@ public extension ReusableView where Self: UITableViewCell {
     init() {
         self.init(style: .default, reuseIdentifier: Self.reuseIdentifier)
 
-        setupLayout()
+        setupSubviews()
```

Comment:
In Xcode, when creating tests, the setup method created is `setUp()`.
But whatever people decide is fine for me.